x-pack/plugins/security_solution/public/common/components/event_details/response_actions_view.tsx

```diff
@@ -71,7 +72,12 @@ export const useResponseActionsView = <T extends object = JSX.Element>({
             ruleName={ruleName}
             ecsData={ecsData}
           />
-        ) : null}
+        ) : (
+          <FormattedMessage
+            id="xpack.securitySolution.eventDetails.responseActionsViewNoActions"
+            defaultMessage="No responses are available for this alert."
```

Comment:
you might want to use [this sentence](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/flyout/right/components/response_button.tsx#L51) (from the new expandable flyout overview section or [this one](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/flyout/left/components/response_details.tsx#L70) (from the new expandable flyout detailed section).
They've both been reviewed and approved by docs. Plus we'd have consistency between the 2 flyouts?

Reply:
@nastasha-solomon what do you think?

Reply:
Sorry, I missed this yesterday @PhilippeOberti. Where's this message appearing? Is it in the right and left panels, or only one place? 

RE the actual message: Some minor edits to make the voice active and to clarify that we're talking about [response actions](https://www.elastic.co/guide/en/security/master/rules-ui-create.html#rule-response-action) only. 

```suggestion
            defaultMessage="This alert does not have response actions."
```

Reply:
@nastasha-solomon unless I'm mistaken this would appear in the left section only (and in the old flyout).
Any reason you would not want to go with one of the options I mentioned in my [comment](https://github.com/elastic/kibana/pull/166916#discussion_r1337364959)? We're already using these options in the new flyout

Reply:
Actually, both of those are perfect. Sorry about that! I must've missed the first comment.

Reply:
@szwarckonrad can you make the change? (see values in my [first comment](https://github.com/elastic/kibana/pull/166916#discussion_r1337364959))

Reply:
Changed.